executor/scripts/build_local.py

```
@@ -0,0 +1,223 @@
+
+    result = subprocess.run(cmd, capture_output=False)
+
+    if result.returncode != 0:
+        print(f"Build failed with return code {result.returncode}")
+        sys.exit(1)
+
+    # Check output
+    output_path = executor_root / "dist" / output_name
+    if output_path.exists():
+        size_mb = output_path.stat().st_size / (1024 * 1024)
+        print(f"\nBuild successful!")
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)